### DISCUSSION TITLE
Simulations for propagation-based phase contrast imaging

### DISCUSSION THREAD

**Discussion by zhanglin-1993** [nrc-cnrc/EGSnrc#689] [Questions] [answered]
Can EGSnrc simulate the propagation-based x-ray phase contrast imaging(PBPCI)? The experiment setup for PBPCI using the microfocus source is similar to that of conventional absortion-based x-ray imaging, except PBPCI considering the spatial coherence of the source. Are there any plans for simulations for PBPCI?

**Comment by rtownson** [comment, marked as answer]
EGSnrc treats photons as particles not waves, so phase contrast mechanisms are not modelled. We don't have plans to go that direction, but EGSnrc is of course open source so anyone could.

**Comment by zhanglin-1993** [comment]
@rtownson Thanks for answering my confusions.

**Comment by ftessier** [comment]
@ilanzhang I recall a talk at the MCTP conference in 2012 reporting on EGSnrc simulation of coherent x-rays. I believe it was Silvia Peter: https://scripts.iucr.org/cgi-bin/paper?hf5246 (and the more recent articles [citing it](https://scholar.google.ca/scholar?cites=1725369169696846673&as_sdt=2005&sciodt=0,5)). The idea is to convert each Monte Carlo photon at a given interface into its plane-wave representation (keeping track of the phase through the optical path length in the MC geometry), and then sum up these wave contribution in small grid cells and further propagate them.

I also note another pertinent reference, where the authors report on their form factor extension to EGSnrc, enabling simulation of diffractive x-ray scattering from powdered crystalline or amorphous materials: https://www.sciencedirect.com/science/article/pii/S0168900207008868.
